tests/foreman/api/test_activationkey.py

```diff
@@ -297,6 +297,8 @@ def setUpClass(cls):
         """Create an organization and an activation key belonging to it."""
         cls.org = entities.Organization().create()
         cls.act_key = entities.ActivationKey(organization=cls.org).create()
+        if rm_bug_is_open(4638):
+            cls.act_key.read()  # Wait for elasticsearch to index new act key.
```

Comment:
I can manually run the failed Jenkins tests and encounter no issues at all. This indicates that whatever issue is causing this test to fail is transient.